cpp/example_code/s3/get_website_config.cpp

```diff
@@ -75,8 +75,10 @@ int main()
     Aws::SDKOptions options;
     Aws::InitAPI(options);
     {
-        Aws::String bucket_name = "my-bucket";
-        Aws::String region = "us-east-1";
+        //TODO: Change to the name of an actual bucket in your account.
+        const Aws::String bucket_name = "DOC-EXAMPLE-BUCKET";
+        //TODO: Set to the region of your profile.

```

Comment:
It's the region in which the bucket was created. Again, I'm not going to point this out from here on.